test/svm/utils.ts

```diff
@@ -74,6 +77,79 @@ export function convertLeafIdToNumber(leaf: RelayerRefundLeafSolana) {
   return { ...leaf, leafId: leaf.leafId.toNumber() };
 }
 
+export function buildRelayerRefundMerkleTree({
+  totalEvmDistributions,
+  totalSolanaDistributions,
+  mixLeaves,
+  chainId,
+  mint,
+  svmRelayers,
+  evmRelayers,
+  evmTokenAddress,
+  evmRefundAmounts,
+  svmRefundAmounts,
+}: {
+  totalEvmDistributions: number;
+  totalSolanaDistributions: number;
+  chainId: number;
+  mixLeaves?: boolean;
+  mint?: PublicKey;
+  svmRelayers?: PublicKey[];
+  evmRelayers?: string[];
+  evmTokenAddress?: string;
+  evmRefundAmounts?: BigNumber[];
+  svmRefundAmounts?: BN[];
+}): { relayerRefundLeaves: RelayerRefundLeafType[]; merkleTree: MerkleTree<RelayerRefundLeafType> } {
+  const relayerRefundLeaves: RelayerRefundLeafType[] = [];
+
+  const createSolanaLeaf = (index: number) => ({
+    isSolana: true,
+    leafId: new BN(index),
+    chainId: new BN(chainId),
+    amountToReturn: new BN(0),
+    mintPublicKey: mint ?? Keypair.generate().publicKey,
+    refundAccounts: svmRelayers || [Keypair.generate().publicKey, Keypair.generate().publicKey],
+    refundAmounts: svmRefundAmounts || [new BN(randomBigInt(2).toString()), new BN(randomBigInt(2).toString())],
+  });
+
+  const createEvmLeaf = (index: number) =>
+    ({
+      isSolana: false,
+      leafId: BigNumber.from(index),
+      chainId: BigNumber.from(chainId),
+      amountToReturn: BigNumber.from(0),
+      l2TokenAddress: evmTokenAddress ?? randomAddress(),
+      refundAddresses: evmRelayers || [randomAddress(), randomAddress()],
+      refundAmounts: evmRefundAmounts || [BigNumber.from(randomBigInt()), BigNumber.from(randomBigInt())],
```

Comment:
it's generally a code smell to have random numbers within unit tests but I think for this it's ok.